data/domain.yml

```diff
@@ -189,14 +189,14 @@ responses:
   utter_services_offline_no_incidents:
     - text: "All services seem to be operating normally."
   utter_services_offline_error:
-    - text: "I was unable to talk with [status.redhat.com](https://status.redhat.com) to fulfill your request."
+    - text: "I was unable to communicate with [status.redhat.com](https://status.redhat.com) to fulfill your request."
```

Comment:
Just a suggestion. Maybe you have a good reason for using 'talk'?

Reply:
Only varying the diction a little bit. It's hard not to use the same words over and over.

Reply:
Sure, but it's easier for users if we are consistent with language. I know it's also boring :-)